subjects/java/projects/travel-plan/audit/README.md

```diff
@@ -0,0 +1,80 @@
+
+#### Comprehension
+
+##### Ask the student to describe the microservices architecture implemented.
+
+###### Did the architecture effectively split functionalities into different microservices based on key factors like scalability and domain separation?
```

Comment:
Maybe add a high-level example of what we mean by this.
```
For example, there is a backend service responsible for reading information on the Postgres and Neo4j DB and providing information to a frontend service that displays the Admin Dashboard defined in the subject 
``` 
Or something similar - for example, are we expecting the final application to be behind a proxy webserver like Nginx or caddy?